cmd/goal/asset.go

```diff
@@ -189,6 +202,46 @@ var createAssetCmd = &cobra.Command{
 		client := ensureFullClient(dataDir)
 		accountList := makeAccountsList(dataDir)
 		creator := accountList.getAddressByName(assetCreator)
+		manager := creator
+		reserve := creator
+		freezer := creator
+		clawback := creator
+
+		if cmd.Flags().Changed("manager") {
+			assetManager = accountList.getAddressByName(assetManager)
+			manager = assetManager
+		}
+
+		if cmd.Flags().Changed("no-manager") {
+			manager = ""
+		}
+
+		if cmd.Flags().Changed("reserve") {
+			assetReserve = accountList.getAddressByName(assetReserve)
+			reserve = assetReserve
+		}
+
+		if cmd.Flags().Changed("no-reserve") {
```

Comment:
A slightly simpler way to use boolean flags is to set the default to false, and then you don't need to use `Changed()`, you can just use the variable directly.  See for example, how `assetFrozen` is used.